metals/src/main/scala/scala/meta/internal/builds/MillBuildTool.scala

```diff
@@ -37,11 +37,23 @@ case class MillBuildTool() extends BuildTool {
       predefScriptPath.toString,
       "mill.contrib.Bloop/install"
     )
+    import scala.collection.JavaConverters._
+    val millVersionPath = workspace.resolve(".mill-version")
+    val millVersion = if (millVersionPath.isFile) {
+      Files
+        .readAllLines(millVersionPath.toNIO)
```

Comment:
does this file contain anything except the one-line version? If no, then you can use `Files.readAllBytes`

Reply:
readAllBytes doesn't give me Strings so this is a bit more useful.

Reply:
There is FileIO.slurp from our internal io libraries.